distributed/tests/test_scheduler.py

```diff
@@ -1367,7 +1367,8 @@ async def test_file_descriptors_dont_leak(s):
 
 @gen_cluster()
 async def test_update_graph_culls(s, a, b):
-    s.update_graph(
+    # FIXME: Migrate to new API
+    s._update_graph_plain_tasks(
         tasks={
             "x": dumps_task((inc, 1)),
```

Comment:
This is the only place where actual functionality of `update_graph` was tested so I kept the original API call around. The functionality this is testing is interesting enough to be preserved since update_graph performs culling as well on top of whatever we're implementing in HLGs, etc.
I chose to keep it for now since I don't know yet what to do with it

cc @rjzamora in case you were not aware. Doesn't change too much about HLGs but this is interesting nonetheless, I believe